Indicators/NormalizedAverageTrueRange.cs

```diff
@@ -0,0 +1,102 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    /// This indicator computes the Normalized Average True Range (NATR).
+    /// The Normalized Average True Range is calculated with the following formula:
+    /// NATR = (ATR(period) / Close) * 100
+    /// </summary>
+    public class NormalizedAverageTrueRange : TradeBarIndicator
+    {
+        private readonly int _period;
+        private readonly TrueRange _tr;
+        private readonly AverageTrueRange _atr;
+        private decimal _lastAtrValue;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NormalizedAverageTrueRange"/> class using the specified name and period.
+        /// </summary> 
+        /// <param name="name">The name of this indicator</param>
+        /// <param name="period">The period of the NATR</param>
+        public NormalizedAverageTrueRange(string name, int period) : 
+            base(name)
+        {
+            _period = period;
+            _tr = new TrueRange(name + "_TR");
+            _atr = new AverageTrueRange(name + "_ATR", period, MovingAverageType.Simple);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NormalizedAverageTrueRange"/> class using the specified period.
+        /// </summary> 
+        /// <param name="period">The period of the NATR</param>
+        public NormalizedAverageTrueRange(int period)
+            : this("NATR" + period, period)
+        {
+        }
+
+        /// <summary>
+        /// Gets a flag indicating when this indicator is ready and fully initialized
+        /// </summary>
+        public override bool IsReady
+        {
+            get { return Samples > _period; }
+        }
+
+        /// <summary>
+        /// Computes the next value of this indicator from the given state
+        /// </summary>
+        /// <param name="input">The input given to the indicator</param>
+        /// <returns>A new value for this indicator</returns>
+        protected override decimal ComputeNextValue(TradeBar input)
+        {
+            _tr.Update(input);
+
+            if (!IsReady)
+            {
+                _atr.Update(input);
+                return _atr / input.Close * 100;
+            }
+
+            if (Samples == _period + 1)
+            {
+                // first output value is SMA of TrueRange
+                _atr.Update(input);
+                _lastAtrValue = _atr;
+            }
+            else
+            {
+                // next TrueRange values are smoothed using Wilder's approach
+                _lastAtrValue = (_lastAtrValue * (_period - 1) + _tr) / _period;
+            }
+
+            return _lastAtrValue / input.Close * 100;
```

Comment:
It's tough. In the divide by zero cases I think we should be returning the result of the limit as the denominator goes to zero. The issue here is technically it goes to infinity, correct? I'm not sure what the right answer is here, I'm open to suggestions, but I feel that zero may be incorrect. @jaredbroad -- any thoughts on limits here?